docs/client.rst

```diff
@@ -0,0 +1,137 @@
+The SecureDrop Client Application
+=================================
+
+As part of the ongoing work to make a user (journalist?) friendly workstation
+for SecureDrop we have created a native client application to be run within the
+Qubes operating system. It helps journalists with the most common activities
+associated with using SecureDrop in a user friendly manner.
+
+Currently the client is alpha quality although work is ongoing in terms of
+improving features and the user interface.
+
+The source code, and related issues are `hosted on GitHub <https://github.com/freedomofpress/securedrop-client>`_.
+
+Developer Setup
+---------------
+
+Clone the repository::
+
+    git clone git@github.com:freedomofpress/securedrop-client.git
+
+Create and enable a new Python3 virtualenv::
+
+    python3 -m venv my_virtualenv
+    source my_virtualenv/bin/activate
+
+Set up the Python dependencies::
+
+    pip install pipenv
+    pipenv install --dev
+    pipenv shell
+
+Install the packaged dependencies::
+
+    sudo apt install python3-pyqt5 python3-pyqt5.qtsvg xvfb
+
+Check you have everything working by running the complete test suite::
+
+    make check
+
+If you see a green report of 100% unit test coverage, you're good to go!
+
+How to Find Help
+----------------
+
+If you would like to report a problem `submit a new issue <https://github.com/freedomofpress/securedrop-client/issues/new>`_.
+
+If you'd like to chat with other developers working on the client drop
+into our `Gitter chat channel for the project <https://gitter.im/freedomofpress/securedrop>`_.
+
+Every week day at 10am (Pacific Time) we take part in a public daily stand-up,
+usually via a `meeting on Jitsi <https://meet.jit.si/QuickWizardsDanceHigh>`_
+(although the details of each daily meeting are published on the Gitter channel
+five minutes before the start of the meeting). All are welcome to contribute.
+
+Otherwise, read on.
+
+Client Architecture
```

Comment:
This feels whole section feels like too much detail outside the client repo. if we want this level of detail, we should make a docs build inside the client that contains this to make it easier to keep things in sync.